framework/generated/generated_vulkan_replay_consumer.cpp

```diff
@@ -5760,11 +5760,11 @@ void VulkanReplayConsumer::Process_vkGetAndroidHardwareBufferPropertiesANDROID(
     uint64_t                                    buffer,
     StructPointerDecoder<Decoded_VkAndroidHardwareBufferPropertiesANDROID>* pProperties)
 {
-    VkDevice in_device = MapHandle<DeviceInfo>(device, &VulkanObjectInfoTable::GetDeviceInfo);
+    auto in_device = GetObjectInfoTable().GetDeviceInfo(device);
     const struct AHardwareBuffer* in_buffer = static_cast<const struct AHardwareBuffer*>(PreProcessExternalObject(buffer, format::ApiCallId::ApiCall_vkGetAndroidHardwareBufferPropertiesANDROID, "vkGetAndroidHardwareBufferPropertiesANDROID"));
-    VkAndroidHardwareBufferPropertiesANDROID* out_pProperties = pProperties->IsNull() ? nullptr : pProperties->AllocateOutputData(1, { VK_STRUCTURE_TYPE_ANDROID_HARDWARE_BUFFER_PROPERTIES_ANDROID, nullptr });
+    pProperties->IsNull() ? nullptr : pProperties->AllocateOutputData(1, { VK_STRUCTURE_TYPE_ANDROID_HARDWARE_BUFFER_PROPERTIES_ANDROID, nullptr });
 
-    VkResult replay_result = GetDeviceTable(in_device)->GetAndroidHardwareBufferPropertiesANDROID(in_device, in_buffer, out_pProperties);
+    VkResult replay_result = OverrideGetAndroidHardwareBufferPropertiesANDROID(GetDeviceTable(in_device->handle)->GetAndroidHardwareBufferPropertiesANDROID, returnValue, in_device, in_buffer, pProperties);
     CheckResult("vkGetAndroidHardwareBufferPropertiesANDROID", returnValue, replay_result);
```

Comment:
Trivia: some of these lines are well over the 132 width limit from the guide (IIRC). Is some hook going to clang-format them?

Reply:
This file is generated, and we don't apply the style guide to generated files.  ([`framework/generated/.clang-format`](https://github.com/LunarG/gfxreconstruct/blob/dev/framework/generated/.clang-format) specifies `DisableFormat`)  You could make an argument that we should, I think?  Maybe that should be a task - to run the generated output through clang-format.  @andrew-lunarg  Would you like to add that task as an issue?

Reply:
I can add that. My take: the formatting would be done in the generator, but still disabled in the `framework/generated/.clang-format` so editors and other tools know to stay out of there.

Reply:
That's a good idea and sounds right.  Thanks!